keyboards/bubble75/hotswap/readme.md

```diff
@@ -0,0 +1,22 @@
+# Bubble 75
+
+![image](https://i.imgur.com/9J01bYH.png)
```

Comment:
```suggestion
![image](https://i.imgur.com/9J01bYHh.png)
```

The additional `h` in the url/filename renders a smaller version of the image, that is still high res but better for github's markdown rendering